config/cc.in

```diff
@@ -53,6 +56,17 @@ config CC_LANG_FORTRAN
       Only select this if you know that your specific version of the
       compiler supports this language.
 
+config CC_LANG_JIT
+    bool
+    prompt "JIT (EXPERIMENTAL)"
+    depends on CC_SUPPORT_JIT
+    depends on EXPERIMENTAL
+    help
+      Enable building the GCC JIT library.
+
+      Only select this if you know that your specific version of the
+      compiler supports this language.
```

Comment:
This probably needs rephrasing since JIT is not a language.

Reply:
You're right, sorry for this oversigh

Reply:
Done!